src/_tests_/SignUp.test.js

```diff
@@ -0,0 +1,34 @@
+import React from "react"
+import { render, screen, fireEvent } from "@testing-library/react"
+import { BrowserRouter } from "react-router-dom"
+import SignUp from "../pages/SignUp"
+import "@testing-library/jest-dom"
+
+const mockSignup = jest.fn()
+
+describe("<SignUp />", () => {
+  beforeEach(() => {
+    mockSignup.mockClear()
+  })
+
+  it("calls signup with correct data on form submission", async () => {
```

Comment:
You are using async here, but I don't see anything using await in the code that follows. Do you need the async here?

Reply:
checking the test now